config/.babelrc.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 				modules: false
 			}
 		],
-		'@babel/preset-stage-0',
+		['@babel/preset-stage-0', {loose: true}],
```

Comment:
I'm not sure it's a good idea to go loose on all of the stage 0 presets.  It's not recommended to use loose, though it will produce faster code... Can we be more targeted?  It seems we only need loose class properties.  See https://github.com/mobxjs/mobx/issues/1352 regarding some babel 7 issues.

Reply:
For the webpack 4 work we are specific and only target the class properties plugin (as presets are gone then). Unless we want larger scale changes in cli, we can take this change as a temporary fix until webpack 4 is merged, or wait until the webpack 4 pr goes in. Moreso upto you guys since it only affects movx for us so far.